packages/server/src/models/objection/User.js

```diff
@@ -50,9 +59,24 @@ export default class User extends BaseModel {
     }
   }
 
+  static async findWithUnverifiedEmail() {
+    const users = await this.query().where({emailVerified: false}).withGraphFetched('[email, emailVerificationToken]')
+    return users.filter(user => !!user.email)
+  }
+
+  async $registeredOnAnj() {
```

Comment:
I’m curious about the naming convention here: does `$` mean that it’s not an static method?

Reply:
In this case I would try to avoid it unless it is extremely necessary

Reply:
okay then, removed it where found one